README.adoc

```diff
@@ -219,6 +219,31 @@ treasury (5 BPS; governable), and updates the user balances with information
 from `DepositSweep.processDepositSweepTxInputs` and
 `DepositSweep.depositSweepTxFeeDistribution`.
 
+==== Redeeming
+
+An account with a balance can request a redemption via
+`Bridge.requestRedemption`. We verify that the destination is valid (P2PKH,
+P2WPKH, P2SH or P2WSH), and build a redemption key based on the wallet's PKH
+and destination. There can only be one pending redemption per PKH-destination
+pair. The treasury takes a cut (`Bridge.redemptionTreasuryFeeDivisor`, 5 BPS),
+and then we reduce the account's balance and start a timer.
+
+After `Bridge.redemptionTimeout` (48 hours), anyone may call
+`Bridge.notifyRedemptionTimeout`. This will decrease
+`wallet.pendingRedemptionsValue`, mark the redemption as "timed out", punish
+the operators for `Bridge.redemptionTimeoutSlashingAmount`, and reward the
+notifier for a percentage (`Bridge.redemptionTimeoutNotifierRewardMultiplier`,
+100%) of the slashed stake. The redeemer is reimbursed the balance of the
```

Comment:
Now I think we should maybe not include 5 BPS, 48h or 100% values in the text at all given they are governable and can get outdated quickly. Having them as default values in the table at the bottom makes sense though.

Reply:
[cdc85fc](https://github.com/keep-network/tbtc-v2/pull/334/commits/cdc85fc2fa5677c6c43d37d358ca258fbb3d53d1)